src/core/ReactContext.js

```diff
@@ -20,7 +20,6 @@ var emptyObject = require('emptyObject');
  * and is accessible via `this.context` on ReactCompositeComponents.
  */
 var ReactContext = {
-
```

Comment:
Don't make this unneccessary change. If you were making any other changes in this file, fine. But you're not.